sdks/python/test-suites/direct/common.gradle

```diff
@@ -162,6 +162,8 @@ tasks.register("directRunnerIT") {
         "apache_beam/io/gcp/bigquery_test.py::BigQueryStreamingInsertTransformIntegrationTests::test_multiple_destinations_transform",
         "apache_beam/io/gcp/bigquery_test.py::PubSubBigQueryIT",
         "apache_beam/io/gcp/bigquery_file_loads_test.py::BigQueryFileLoadsIT::test_bqfl_streaming",
+        "apache_beam/io/gcp/bigquery_file_loads_test.py::BigQueryFileLoadsIT::test_bqfl_streaming_with_dynamic_destinations",
+        "apache_beam/io/gcp/bigquery_file_loads_test.py::BigQueryFileLoadsIT::test_one_job_fails_all_jobs_fail",
```

Comment:
If I understood correctly this line should refer to `test_bqfl_streaming_with_copy_jobs` (see skipped test in https://ci-beam.apache.org/job/beam_PostCommit_Python37_PR/444/testReport/apache_beam.io.gcp.bigquery_file_loads_test/BigQueryFileLoadsIT/)

Reply:
Fixed. Good catch